src/main/java/core/basesyntax/model/Operation.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+public enum Operation {
+    BALANCE("b"),
+    SUPPLY("s"),
+    PURCHASE("p"),
+    RETURN("r");
+
+    private final String oneLetter;
+
+    Operation(String oneLetter) {
+        this.oneLetter = oneLetter;
+    }
+
+    public String getOneLetter() {
+        return oneLetter;
+    }
+
+    public static Operation of(String oneLetter) {
+        switch (oneLetter) {
+            case "b" : return Operation.BALANCE;
+            case "s" : return Operation.SUPPLY;
+            case "p" : return Operation.PURCHASE;
+            default : return Operation.RETURN;
+        }
```

Comment:
Lets implement this functionality with loop (or Map if you want to speed up this solution), so we wont need to add more case clauses if we add some new Operation in the future